api/src/main/java/org/pmiops/workbench/reporting/ReportingUploadServiceImpl.java

```diff
@@ -122,8 +151,7 @@ private void checkResponseAndRowCounts(
   }
 
   private List<InsertAllRequest> getInsertAllRequests(ReportingSnapshot reportingSnapshot) {
```

Comment:
deprecate this method and annotate that it's only used in the snapshot (memory-wasting) implementation.